tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc

```diff
@@ -2149,7 +1967,26 @@ TEST_F(OpConverterTest, ConvertBinary) {
   TestBinaryTensorOpTensor<ops::Minimum, DT_FLOAT>(this);
   TestBinaryTensorOpTensor<ops::Maximum, DT_FLOAT>(this);
   TestBinaryTensorOpTensor<ops::Pow, DT_FLOAT>(this);
-
+  // Test with operand R = Weights
```

Comment:
nit: it would be good to use templates to avoid duplicating these calls for different function/dtypes. I think we can do something like https://stackoverflow.com/a/4697356/4691962.